Source/PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>NSPrivacyCollectedDataTypes</key>
+	<array>
+		<dict>
+			<key>NSPrivacyCollectedDataType</key>
+			<string>NSPrivacyCollectedDataTypeDeviceID</string>
+			<key>NSPrivacyCollectedDataTypeLinked</key>
+			<true/>
+			<key>NSPrivacyCollectedDataTypeTracking</key>
+			<true/>
+			<key>NSPrivacyCollectedDataTypePurposes</key>
+			<array>
+				<string>NSPrivacyCollectedDataTypePurposeAnalytics</string>
+				<string>NSPrivacyCollectedDataTypePurposeAppFunctionality</string>
+				<string>NSPrivacyCollectedDataTypePurposeThirdPartyAdvertising</string>
```

Comment:
I think the adapter is using the IDFA only to enable test mode? In that case maybe the purpose should just be `NSPrivacyCollectedDataTypePurposeAppFunctionality`?
In Apple's documentation, AppFunctionality purpose includes "enable features", which makes me thing enabling the test mode feature can be included in this purpose

Reply:
Let's keep this PR open until we hear back from Legal and PM. Thanks!

Reply:
Slack discussion: https://chartboost.slack.com/archives/C05KX3QRQ0L/p1699377956737459